aspnetcore/tutorials/first-mongo-app.md

```diff
@@ -49,11 +49,18 @@ In this tutorial, you learn how to:
 
 ## Configure MongoDB
 
-Enable MongoDB and Mongo DB Shell access from anywhere on the development machine:
 
-1. On Windows, MongoDB is installed at *C:\\Program Files\\MongoDB* by default. Add *C:\\Program Files\\MongoDB\\Server\\\<version_number>\\bin* to the `PATH` environment variable.
-1. Download the MongoDB Shell and choose a directory to extract it to. Add the resulting path for `mongosh.exe` to the `PATH` environment variable.
-1. Choose a directory on the development machine for storing the data. For example, *C:\\BooksData* on Windows. Create the directory if it doesn't exist. The mongo Shell doesn't create new directories.
+Enable MongoDB and Mongo DB Shell access from anywhere on the development machine (Windows/Linux/macOS):
+
+1. Download and Install MongoDB Shell:
+   * macOS/Linux: Choose a directory to extract the MongoDB Shell to. Add the resulting path for `mongosh` to the `PATH` environment variable.
```

Comment:
One spot has Mongo DB Shell, the other has MongoDB Shell

Reply:
Will fix.  Should be MongoDB Shell.